### PR TITLE
Add SafeNet MobilePASS

### PR DESCRIPTION
**Pull request by Razzeee**
### Description

This should enable SafeNet MobilePASS to be downloaded and run. 

### What works

Dosen't work.

```
[klampe@nbssc150os ~]$ flatpak run org.phoenicis.playonlinux 
Gtk-Message: 19:37:07.079: Failed to load module "canberra-gtk-module"
Gtk-Message: 19:37:07.079: Failed to load module "pk-gtk-module"
Gtk-Message: 19:37:07.079: Failed to load module "canberra-gtk-module"
Gtk-Message: 19:37:07.079: Failed to load module "pk-gtk-module"
ERROR StatusLogger No Log4j 2 configuration file found. Using default configuration (logging only errors to the console), or user programmatically provided configurations. Set system property 'log4j2.debug' to show Log4j 2 internal initialization logging. See https://logging.apache.org/log4j/2.x/manual/configuration.html for instructions on how to configure Log4j 2
[INFO ] org.phoenicis.repository.DefaultRepositoryManager (l.133) - Adding repositories: [org.phoenicis.repository.location.GitRepositoryLocation@5ce1c68[https://github.com/Razzeee/scripts,add-mobilepass], org.phoenicis.repository.location.GitRepositoryLocation@476c8784[https://github.com/PhoenicisOrg/scripts,release/5.0-alpha-3], org.phoenicis.repository.location.ClasspathRepositoryLocation@1d19bd3e[/org/phoenicis/repository]] at index 0
Jan 30, 2021 7:37:11 PM com.sun.javafx.css.StyleManager$ImageCache getCachedImage
WARNING: Error loading image: file:/home/klampe/.Phoenicis/shortcuts/MobilePass.miniature
Jan 30, 2021 7:37:13 PM com.sun.javafx.css.StyleManager$ImageCache getCachedImage
WARNING: Error loading image: file:/home/klampe/.Phoenicis/shortcuts/MobilePass.miniature
[INFO ] org.phoenicis.repository.types.MultipleRepository (l.46) - Fetching applications for: org.phoenicis.repository.types.MultipleRepository@db726ca[repositories=[GitRepository(url: 'https://github.com/Razzeee/scripts', folder: '/home/klampe/.Phoenicis/cache/git1643601134', branch: 'add-mobilepass'), GitRepository(url: 'https://github.com/PhoenicisOrg/scripts', folder: '/home/klampe/.Phoenicis/cache/git-298283779', branch: 'release/5.0-alpha-3'), org.phoenicis.repository.types.ClasspathRepository@370d90c[packagePath=/org/phoenicis/repository]]]
[INFO ] org.phoenicis.repository.types.GitRepository (l.72) - Begin fetching process of 'https://github.com/PhoenicisOrg/scripts' to '/home/klampe/.Phoenicis/cache/git-298283779'
[INFO ] org.phoenicis.repository.types.GitRepository (l.135) - Opening git-repository at '/home/klampe/.Phoenicis/cache/git-298283779'
Jan 30, 2021 7:37:16 PM com.sun.javafx.css.StyleManager$ImageCache getCachedImage
WARNING: Error loading image: file:/home/klampe/.Phoenicis/shortcuts/MobilePass.miniature
[INFO ] org.phoenicis.repository.types.GitRepository (l.141) - Pulling new commits to '/home/klampe/.Phoenicis/cache/git-298283779'
[INFO ] org.phoenicis.repository.types.GitRepository (l.145) - Finished pulling new commits to '/home/klampe/.Phoenicis/cache/git-298283779'
[INFO ] org.phoenicis.repository.types.GitRepository (l.72) - Begin fetching process of 'https://github.com/Razzeee/scripts' to '/home/klampe/.Phoenicis/cache/git1643601134'
[INFO ] org.phoenicis.repository.types.GitRepository (l.135) - Opening git-repository at '/home/klampe/.Phoenicis/cache/git1643601134'
[INFO ] org.phoenicis.repository.types.LocalRepository (l.77) - Reading directory : /home/klampe/.Phoenicis/cache/git-298283779
[INFO ] org.phoenicis.repository.types.GitRepository (l.141) - Pulling new commits to '/home/klampe/.Phoenicis/cache/git1643601134'
[INFO ] org.phoenicis.repository.types.GitRepository (l.145) - Finished pulling new commits to '/home/klampe/.Phoenicis/cache/git1643601134'
[INFO ] org.phoenicis.repository.types.LocalRepository (l.77) - Reading directory : /home/klampe/.Phoenicis/cache/git1643601134
[Use -Dgraal.LogFile=<path> to redirect Graal log output to a file.]
Cannot install GraalVM MBean due to Receiver class org.graalvm.compiler.hotspot.management.HotSpotGraalManagement does not define or inherit an implementation of the resolved method 'abstract void initialize(org.graalvm.compiler.hotspot.HotSpotGraalRuntime, org.graalvm.compiler.hotspot.GraalHotSpotVMConfig)' of interface org.graalvm.compiler.hotspot.HotSpotGraalManagementRegistration.
Jan 30, 2021 7:37:21 PM com.sun.javafx.css.StyleManager$ImageCache getCachedImage
WARNING: Error loading image: file:/home/klampe/.Phoenicis/shortcuts/MobilePass.miniature
Jan 30, 2021 7:37:23 PM com.sun.javafx.css.StyleManager$ImageCache getCachedImage
WARNING: Error loading image: file:/home/klampe/.Phoenicis/shortcuts/MobilePass.miniature
[WARN ] org.phoenicis.javafx.views.mainwindow.installations.dto.InstallationDTO (l.56) - Installation ID (SafeNet MobilePASS_1612031850628) contains invalid characters, will remove them.
[ERROR] org.phoenicis.multithreading.ControlledThreadPoolExecutorService (l.64) - Unable to create temporary file, phoenicis2534296686935199040.com/filesSN/Index/79cca6a24fcc41c48a6dc31162a9ee7e?sid=6aded7501b842c50f12064606e4bcbaf
	at java.base/java.io.File$TempDirectory.generateFile(File.java:1986)
	at java.base/java.io.File.createTempFile(File.java:2078)
	at phoenicis.tools@5.0-alpha-3/org.phoenicis.tools.files.FileUtilities.createTmpFile(FileUtilities.java:270)
	at <js> createTempFile(Unnamed:137:2980-3017)
	at <js> _installationCommand(Unnamed:39:1158-1182)
	at <js> go(Unnamed:29:1044-1081)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.ObjectProxyHandler.invoke(HostInteropReflect.java:694)
	at com.sun.proxy.$Proxy64.go(Unknown Source)
	at phoenicis.javafx@5.0-alpha-3/org.phoenicis.javafx.components.application.skin.ApplicationInformationPanelSkin.lambda$installScript$7(ApplicationInformationPanelSkin.java:271)
	at phoenicis.scripts@5.0-alpha-3/org.phoenicis.scripts.session.PhoenicisInteractiveScriptSession.eval(PhoenicisInteractiveScriptSession.java:35)
	at phoenicis.scripts@5.0-alpha-3/org.phoenicis.scripts.interpreter.BackgroundScriptInterpreter.lambda$createInteractiveSession$1(BackgroundScriptInterpreter.java:45)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by host exception: java.io.IOException: Unable to create temporary file, phoenicis2534296686935199040.com/filesSN/Index/79cca6a24fcc41c48a6dc31162a9ee7e?sid=6aded7501b842c50f12064606e4bcbaf

Exception in thread "pool-4-thread-2" Unable to create temporary file, phoenicis2534296686935199040.com/filesSN/Index/79cca6a24fcc41c48a6dc31162a9ee7e?sid=6aded7501b842c50f12064606e4bcbaf
	at java.base/java.io.File$TempDirectory.generateFile(File.java:1986)
	at java.base/java.io.File.createTempFile(File.java:2078)
	at phoenicis.tools@5.0-alpha-3/org.phoenicis.tools.files.FileUtilities.createTmpFile(FileUtilities.java:270)
	at <js> createTempFile(Unnamed:137:2980-3017)
	at <js> _installationCommand(Unnamed:39:1158-1182)
	at <js> go(Unnamed:29:1044-1081)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.ObjectProxyHandler.invoke(HostInteropReflect.java:694)
	at com.sun.proxy.$Proxy64.go(Unknown Source)
	at phoenicis.javafx@5.0-alpha-3/org.phoenicis.javafx.components.application.skin.ApplicationInformationPanelSkin.lambda$installScript$7(ApplicationInformationPanelSkin.java:271)
	at phoenicis.scripts@5.0-alpha-3/org.phoenicis.scripts.session.PhoenicisInteractiveScriptSession.eval(PhoenicisInteractiveScriptSession.java:35)
	at phoenicis.scripts@5.0-alpha-3/org.phoenicis.scripts.interpreter.BackgroundScriptInterpreter.lambda$createInteractiveSession$1(BackgroundScriptInterpreter.java:45)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by host exception: java.io.IOException: Unable to create temporary file, phoenicis2534296686935199040.com/filesSN/Index/79cca6a24fcc41c48a6dc31162a9ee7e?sid=6aded7501b842c50f12064606e4bcbaf

```

### What was not tested

As I can't get past that error - it keeps installing, not much was tested.

### Test
- Operating system (and linux kernel version): Fedora 33
- Hardware (GPU/CPU): Lenovo T14 (Amd Mobile CPU)

### Ready for review
- [x] Script tested as a regular phoenicis user and working (if you have a problem -> create as draft and ask for help).
- [x] `json-align` and `eslint` run according to the [documentation](https://phoenicisorg.github.io/scripts/General/tools/). 
- [ ] Codacy and travis checked.

### CODE REVIEW

**Comment by plata** [comment]
Please merge master (there have been some changes).

**Comment by Razzeee** [comment]
Rebased

File: Applications/Accessories/MobilePASS/Online/script.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const OnlineInstallerScript = include("engines.wine.quick_script.online_installer_script");
+
+new OnlineInstallerScript()
+    .name("SafeNet MobilePASS")
+    .editor("SafeNet")
+    .applicationHomepage("https://safenetmp.com/")
+    .author("Kolja Lampe")
+    .url("https://fdsext.gemalto.com/filesSN/Index/79cca6a24fcc41c48a6dc31162a9ee7e?sid=6aded7501b842c50f12064606e4bcbaf")
```
Comment (plata) [inline Applications/Accessories/MobilePASS/Online/script.js:8]:
```suggestion
    .url("https://fdsext.gemalto.com/filesSN/Index/79cca6a24fcc41c48a6dc31162a9ee7e?sid=6aded7501b842c50f12064606e4bcbaf.exe")
```
The problem is that we try to infer the extension:
https://github.com/PhoenicisOrg/phoenicis/blob/dc91631f4b0e03142952d761600838de6cb53cb8/phoenicis-tools/src/main/java/org/phoenicis/tools/files/FileUtilities.java#L270

File: Applications/Accessories/MobilePASS/Online/script.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    .checksum("b804feb434dd7fbed5f73508279f2362b3d1badd")
+    .category("Accessories")
+    .executable("MobilePASS.exe");
```
Comment (plata) [inline Applications/Accessories/MobilePASS/Online/script.js:11]: Add new line at EOF.

**Review by plata — CHANGES_REQUESTED** [review]
We need a proper `main.png` before we can merge this.

**Comment by Razzeee** [comment]
This seems to make downloading work, still it's not in a functional state.

As the exe is not a setup, the script still thinks it's installing, even after closing the apps popup. And after closing playOnLinux the `Applications` tab is empty again, probably due to the "setup" never finishing?

**Comment by plata** [comment]
In this case, you probably want more control over the installation such that you can just copy the downloaded exe into the prefix and set the shortcut. Take a look at the script for [IE 6.0](https://github.com/PhoenicisOrg/scripts/blob/master/Applications/Internet/Internet%20Explorer%206.0/Online/script.js). Don't get overwhelmed at first glance. You will not need the DLL overrides etc. so it will boil down to quite a little amount of code.

**Comment by Razzeee** [comment]
Not sure what I missed, as linting seems fine, but the error when trying to run it isn't very helpful:

```
SyntaxError: Unnamed:7:0 Expected an operand but found <<
<<<<<<< HEAD
^
Unnamed:65:0 Expected an operand but found ===
=======
^
Unnamed:119:0 Expected an operand but found >>>
>>>>>>> branch 'add-mobilepass' of https://github.com/Razzeee/scripts
^
Unnamed:131:0 Expected } but found eof

^

	at org.graalvm.sdk/org.graalvm.polyglot.Context.eval(Context.java:371)
	at phoenicis.scripts@5.0-alpha-3/org.phoenicis.scripts.engine.implementation.PolyglotScriptEngine.evalAndReturn(PolyglotScriptEngine.java:73)
	at phoenicis.scripts@5.0-alpha-3/org.phoenicis.scripts.session.PhoenicisInteractiveScriptSession.eval(PhoenicisInteractiveScriptSession.java:35)
	at phoenicis.scripts@5.0-alpha-3/org.phoenicis.scripts.interpreter.BackgroundScriptInterpreter.lambda$createInteractiveSession$1(BackgroundScriptInterpreter.java:45)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

```

**Comment by plata** [comment]
Just tried myself and I get:
```
Caused by host exception: java.lang.IllegalArgumentException: The file (MobilePass/MobilePASS.exe) must be in the Phoenicis root directory (/home/<user>/.Phoenicis)
```

File: Applications/Accessories/MobilePASS/Online/script.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            "MobilePass",
+            "upstream",
+            "x86",
+            getLatestStableVersion(setupWizard, "x86")
+        )
+        .create();
+
+    new WindowsVersion(wine).withWindowsVersion("winvista").go();
+
+    cp(tempFile, "MobilePass/MobilePASS.exe");
```
Comment (plata) [inline Applications/Accessories/MobilePASS/Online/script.js:53]:
```suggestion
    cp(tempFile, wine.prefixDirectory() + "MobilePass/MobilePASS.exe");
```

**Comment by Razzeee** [comment]
Doesn't seem like copy works

```
[WARN ] org.phoenicis.javafx.views.mainwindow.installations.dto.InstallationDTO (l.56) - Installation ID (SafeNet MobilePASS_1613034949435) contains invalid characters, will remove them.
[ERROR] org.phoenicis.multithreading.ControlledThreadPoolExecutorService (l.64) - /home/klampe/.Phoenicis/containers/wineprefix/MobilePass/drive_c/MobilePass/MobilePASS.exe (No such file or directory)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
	at phoenicis.tools@5.0-alpha-3/org.phoenicis.tools.win32.ExeAnalyser.isArchitecture(ExeAnalyser.java:61)
	at phoenicis.tools@5.0-alpha-3/org.phoenicis.tools.win32.ExeAnalyser.is64Bits(ExeAnalyser.java:46)
	at <js> run(Unnamed:407:15350-15396)
	at <js> run(Unnamed:172:5961-6051)
	at <js> runInsidePrefix(Unnamed:139:4721-4822)
	at <js> :=>(Unnamed:59:1794-1852)
	at <js> go(Unnamed:25:594-607)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.ObjectProxyHandler.invoke(HostInteropReflect.java:694)
	at com.sun.proxy.$Proxy63.go(Unknown Source)
	at phoenicis.javafx@5.0-alpha-3/org.phoenicis.javafx.components.application.skin.ApplicationInformationPanelSkin.lambda$installScript$7(ApplicationInformationPanelSkin.java:271)
	at phoenicis.scripts@5.0-alpha-3/org.phoenicis.scripts.session.PhoenicisInteractiveScriptSession.eval(PhoenicisInteractiveScriptSession.java:35)
	at phoenicis.scripts@5.0-alpha-3/org.phoenicis.scripts.interpreter.BackgroundScriptInterpreter.lambda$createInteractiveSession$1(BackgroundScriptInterpreter.java:45)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by host exception: java.io.FileNotFoundException: /home/klampe/.Phoenicis/containers/wineprefix/MobilePass/drive_c/MobilePass/MobilePASS.exe (No such file or directory)

Exception in thread "pool-4-thread-2" /home/klampe/.Phoenicis/containers/wineprefix/MobilePass/drive_c/MobilePass/MobilePASS.exe (No such file or directory)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
	at phoenicis.tools@5.0-alpha-3/org.phoenicis.tools.win32.ExeAnalyser.isArchitecture(ExeAnalyser.java:61)
	at phoenicis.tools@5.0-alpha-3/org.phoenicis.tools.win32.ExeAnalyser.is64Bits(ExeAnalyser.java:46)
	at <js> run(Unnamed:407:15350-15396)
	at <js> run(Unnamed:172:5961-6051)
	at <js> runInsidePrefix(Unnamed:139:4721-4822)
	at <js> :=>(Unnamed:59:1794-1852)
	at <js> go(Unnamed:25:594-607)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.ObjectProxyHandler.invoke(HostInteropReflect.java:694)
	at com.sun.proxy.$Proxy63.go(Unknown Source)
	at phoenicis.javafx@5.0-alpha-3/org.phoenicis.javafx.components.application.skin.ApplicationInformationPanelSkin.lambda$installScript$7(ApplicationInformationPanelSkin.java:271)
	at phoenicis.scripts@5.0-alpha-3/org.phoenicis.scripts.session.PhoenicisInteractiveScriptSession.eval(PhoenicisInteractiveScriptSession.java:35)
	at phoenicis.scripts@5.0-alpha-3/org.phoenicis.scripts.interpreter.BackgroundScriptInterpreter.lambda$createInteractiveSession$1(BackgroundScriptInterpreter.java:45)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by host exception: java.io.FileNotFoundException: /home/klampe/.Phoenicis/containers/wineprefix/MobilePass/drive_c/MobilePass/MobilePASS.exe (No such file or directory)


```

**Comment by plata** [comment]
Could be that you must create the `MobilePass` directory first with `mkdir`.

**Comment by Razzeee** [comment]
Doesn't seem to help

**Comment by plata** [comment]
The problem is that `prefixDirectory()` returns really the prefix, see
https://github.com/PhoenicisOrg/scripts/blob/29af2899ed9704f4d76056afbb7bace48e87e9d2/Engines/Wine/Verbs/xact/script.js#L41
but `runInsidePrefix()` runs in `drive_c`, see
https://github.com/PhoenicisOrg/scripts/blob/29af2899ed9704f4d76056afbb7bace48e87e9d2/Engines/Wine/Engine/Object/script.js#L128

To be honest, I cannot explain why it is like that. @madoar @qparis can you explain?

Try:
```JS
cp(tempFile, wine.prefixDirectory() + "/drive_c/MobilePass/MobilePASS.exe");
```

**Comment by Razzeee** [comment]
Setup is a bit weird, as it keeps on showing the progess, while the app already runs.
Peek 2021-02-15 18-05

Otherwise it seems to exactly do what I want it to do now :)

**Comment by Razzeee** [comment]
Removing `wine.runInsidePrefix("MobilePass/MobilePASS.exe", [], true);` is fine, works like it should now. The only weird thing is, that after installation it opens up the "Installations" tab, which will be empty and you need to change to the "Library" tab instead.

If your happy otherwise, I can probably try to somehow generate a picture and squash the code to one commit. Let me know.

**Comment by plata** [comment]
You don't need to squash. I will squash merge anyways when ready. We need a picture and you should merge master. Then I can do the review.

**Comment by Razzeee** [comment]
Should hopefully be good now

File: Applications/Accessories/MobilePASS/Online/script.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        .wizard(setupWizard)
+        .prefix(
+            "MobilePass",
+            "upstream",
+            "x86",
+            getLatestStableVersion(setupWizard, "x86")
+        )
+        .create();
+
+    new WindowsVersion(wine).withWindowsVersion("winxp").go();
```
Comment (plata) [inline Applications/Accessories/MobilePASS/Online/script.js:51]: Required to use winxp?

**Comment by plata** [comment]
There was a bug in the actions such that they didn't run for PRs from forks. Please merge master again.

File: Applications/Accessories/MobilePASS/Online/script.js (hunk 4 above)
Comment (Razzeee) [reply Applications/Accessories/MobilePASS/Online/script.js:51]: No, what should be the default? I just choose this, as it's what the frontend defaults to when doing a local install.

**Comment by Razzeee** [comment]
Rebased

**Comment by plata** [comment]
The jsons must be fixed. `id` is missing.

**Comment by Razzeee** [comment]
Fixed

**Comment by plata** [comment]
I don't know what happened here but something broke the GitHub actions. Maybe the force push?

**Comment by Razzeee** [comment]
@plata `npm run json-align` found that it wanted `settings.json` to be formatted differently, but couldn't push that. I've done those changes on my end now. Translation seems to have a similar problem.

File: Applications/Accessories/MobilePASS/Online/script.js (hunk 4 above)
Comment (Razzeee) [reply Applications/Accessories/MobilePASS/Online/script.js]: Switched to win7 now, will need to retest

**Comment by plata** [comment]
Ok, the whole GitHub actions setup is quite new and still not completely stabilized. I've updated the actions to run the linter and the translations update only on master. Would you mind merging master again? Sorry for the inconvenience.

**Comment by Razzeee** [comment]
Thank you, updated

File: .vscode/settings.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+    "prettier.enable" : false
```
Comment (plata) [inline .vscode/settings.json:2]: Did you commit this on purpose?

File: Applications/Accessories/MobilePASS/Online/script.js [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+        .wizard(setupWizard)
+        .prefix(
+            "MobilePass",
+            "upstream",
+            "x86",
+            getLatestStableVersion(setupWizard, "x86")
+        )
+        .create();
+
+    new WindowsVersion(wine).withWindowsVersion("win7").go();
```
Comment (plata) [inline Applications/Accessories/MobilePASS/Online/script.js:51]: If there's no special requirement for the Windows version, you can just leave this away completely.

File: Applications/Accessories/MobilePASS/Online/script.js [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+    mkdir(wine.prefixDirectory() + "MobilePass");
```
Comment (plata) [inline Applications/Accessories/MobilePASS/Online/script.js:53]: Is the `mkdir` needed or was I wrong?

File: i18n/Messages.properties [hunk 8]
```diff
@@ -328,15 +328,16 @@ DXVK=DXVK
 #: Engines/Wine/Verbs/DXVK/script.js:41
 DXVK\ is\ currently\ unsupported\ on\ non-Linux\ operating\ systems\ due\ to\ MoltenVK\ implementation\ being\ incomplete.\ Select\ how\ do\ you\ want\ to\ approach\ this\ situation.=DXVK is currently unsupported on non-Linux operating systems due to MoltenVK implementation being incomplete. Select how do you want to approach this situation.
 
-#: Engines/Wine/Settings/GLSL/script.js:11 Engines/Wine/Settings/render target
-#: lock mode/script.js:11 Engines/Wine/Settings/DirectDraw
-#: renderer/script.js:11 Engines/Wine/Settings/video memory size/script.js:11
-#: Engines/Wine/Settings/mouse warp override/script.js:11
+#: Engines/Wine/Settings/offscreen rendering mode/script.js:11
```
Comment (plata) [inline i18n/Messages.properties:331]: Can you revert the changes in this file? Not sure if it might cause problems.

**Review by plata — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 5 above)
Comment (Razzeee) [reply .vscode/settings.json:2]: This was done by the json align, but I can revert it, like you suggest for the translation stuff.

Comment (Razzeee) [reply .vscode/settings.json:2]: I'm wrong, it slipped in by mistake

File: Applications/Accessories/MobilePASS/Online/script.js (hunk 6 above)
Comment (Razzeee) [reply Applications/Accessories/MobilePASS/Online/script.js:51]: Removed

File: Applications/Accessories/MobilePASS/Online/script.js (hunk 7 above)
Comment (Razzeee) [reply Applications/Accessories/MobilePASS/Online/script.js:53]: I think it's needed, it didn't work before.

File: i18n/Messages.properties (hunk 8 above)
Comment (Razzeee) [reply i18n/Messages.properties]: Done

**Review by plata — APPROVED** [review]
